misc/avoid_warning_linux.py

```diff
@@ -0,0 +1,33 @@
+import fileinput
+import os
+import re
+import sys
+import getpass
+
+def comment_lines(file_path, lines_to_comment):
+    with fileinput.FileInput(file_path, inplace=True, backup='.bak') as file:
+        for line_number, line in enumerate(file, start=1):
+            if line_number in lines_to_comment:
+                print(f"# {line}", end='')
+            else:
+                print(line, end='')
+
+
+# Get the username of the current user
+username = getpass.getuser()
+# Get the user's Python version
+python_version = f"{sys.version_info.major}.{sys.version_info.minor}"
+# Construct the path to the script based on the username and Python version
+script_file = f"/home/{username}/anaconda3/envs/brainchulo/lib/python{python_version}/site-packages/chromadb/api/models/Collection.py"
```

Comment:
we should not assume everyone uses conda.

Reply:
let's ponder whether this file is really needed.

Reply:
Yeah I'll remove it from the PR, the loggerhandler solution seems much better